components/navbar.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+import Headroom from 'react-headroom';
+import { prefixLink } from 'gatsby-helpers';
```

Comment:
not 100% sure what ur dependency error is, but you might need a `// eslint-disable-line` if it's not recognizing gatsy-helpers (I had an error w this too and this fixed it--at least temporarily)